src/nedap/utils/test/impl.cljc

```diff
@@ -53,10 +53,16 @@
 #?(:cljs
    (do
      (derive ::exit-code-reporter :cljs.test/default)
+
+     (defn set-exit-code-for! [summary process-object]
+       (assert (map? summary))
+       (if-not (cljs.test/successful? summary)
+         (set! (.-exitCode process-object) 1)
+         (when-not (.-exitCode process-object)
+           (set! (.-exitCode process-object) 0))))
```

Comment:
This looks a bit funny due to the nested if/when's , but using smt like `cond` doesn't _really_ improve on it.

```clj
(cond 
  (not (cljs.test/successful? summary))
  (set! (.-exitCode process-object) 1)
  
  (and (cljs.test/successful? summary)
       (not (.-exitCode process-object)))
  (set! (.-exitCode process-object) 0)))
```

fine to keep as is.